From ea144cdaf7bd478d090fd9dc20ccc256d2385a38 Mon Sep 17 00:00:00 2001
From: Zorro Liu <lyx@rock-chips.com>
Date: Thu, 12 Nov 2020 22:03:30 +0800
Subject: [PATCH] arm64: configs: rockchip_defconfig: add wacom_9013 and
 cyttsp5 touch drivers

use on rk3566-evb2-lp4x-v10-eink board

Signed-off-by: Zorro Liu <lyx@rock-chips.com>
Change-Id: Ibd9cd9a3001393a2103eae4f8f45f7416d73fabc
---
 arch/arm64/configs/rockchip_defconfig | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index c52792d1a0f3..ade72f2aa97b 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -441,8 +441,14 @@ CONFIG_TOUCHSCREEN_GSL3673_800X1280=y
 CONFIG_TOUCHSCREEN_GSL3676=y
 CONFIG_TOUCHSCREEN_GT9XX=y
 CONFIG_TOUCHSCREEN_HYN_CST2XX=y
+CONFIG_TOUCHSCREEN_WACOM_W9013=y
 CONFIG_TOUCHSCREEN_GT1X=y
 CONFIG_TOUCHSCREEN_FTS=y
+CONFIG_TOUCHSCREEN_CYPRESS_CYTTSP5=y
+CONFIG_TOUCHSCREEN_CYPRESS_CYTTSP5_DEVICETREE_SUPPORT=y
+CONFIG_TOUCHSCREEN_CYPRESS_CYTTSP5_I2C=y
+CONFIG_TOUCHSCREEN_CYPRESS_CYTTSP5_DEVICE_ACCESS=y
+CONFIG_TOUCHSCREEN_CYPRESS_CYTTSP5_LOADER=y
 CONFIG_ROCKCHIP_REMOTECTL=y
 CONFIG_ROCKCHIP_REMOTECTL_PWM=y
 CONFIG_SENSOR_DEVICE=y
-- 
2.35.3

